src/chainparams.h

```diff
@@ -43,6 +43,7 @@ class CChainParams
         EXT_SECRET_KEY,
 
         ZCPAYMENT_ADDRRESS,
+        ZCVIEWING_KEY,
```

Comment:
Why is ZCVIEWING_KEY in between, rather than at end?
It will change the enum value of ZCSPENDING_KEY - is that safe?

Reply:
Hmm, good point. Probably best to be safe and swap them around.